pkg/nsx/services/common/store.go

```diff
@@ -85,3 +82,42 @@ func TransError(err error) error {
 	}
 	return err
 }
+
+// QueryResource is the method to query all the various resources from nsx-t side,
+// we could use it to cache all the resources in the store when process starts.
+func (service *Service) QueryResource(wg *sync.WaitGroup, fatalErrors chan error, resourceTypeValue string, store Store) {
```

Comment:
we only see this func QueryResource called during InitializeSecurityPolicy, that is, to Initialize stores related to SecurityPolicy, do we have other usage for this func?

Reply:
If just for initializing the store resource, maybe initResourceStore?

Reply:
ok